libs/core/src/components/alert/alert.component.ts

```diff
@@ -0,0 +1,273 @@
+import { nothing, PropertyValues } from 'lit'
+import { property, query, state } from 'lit/decorators.js'
+import { classMap } from 'lit/directives/class-map.js'
+import { createRef, ref, Ref } from 'lit/directives/ref.js'
+import { html as staticHtml, unsafeStatic } from 'lit/static-html.js'
+
+import { GdsElement } from '../../gds-element'
+import { gdsCustomElement, html } from '../../scoping'
+import { GdsButton } from '../button/button.component'
+import { GdsCard } from '../card/card.component'
+
+import '../icon/icons/checkmark.js'
+import '../icon/icons/triangle-exclamation.js'
+import '../icon/icons/circle-info.js'
+import '../icon/icons/cross-small.js'
+import '../button/index.js'
+
+import { tokens } from '../../tokens.style'
+import { alertStyles } from './alert.style'
+
+// Type definitions
+type AlertVariant =
```

Comment:
@splashdust since these types are used a little bit everywhere throughout the components, are they typed somewhere else and can be inherited?

If not, I think these should be exported also @yuyo2